model/src/main/proto/javaclasses/mealorder/c/rejections.proto

```diff
@@ -47,113 +48,116 @@ message VendorAlreadyExists {
     // A name of existing vendor.
     VendorName vendor_name = 2;
 
-    // A Date when a command was rejected.
+    // A time point when it was rejected.
     google.protobuf.Timestamp when_rejected = 3;
 }
 
 // The rejection to set date range for menu.
 //
-// It is thrown if the menu for this period already exists or the range includes dates from the past.
+// It is thrown if a menu for this period already exists or a range includes dates from the past.
 //
 message CannotSetDateRange {
 
-    // An identifier of the vendor of this menu.
+    // An identifier of a vendor of this menu.
     VendorId vendor_id = 1;
 
-    // An identifier of the menu for which no date is set.
+    // An identifier of a menu for which no date is set.
     MenuId menu_id = 2;
 
     // A date range that was been attempted to set.
     MenuDateRange menu_date_range = 3;
 
-    // A Date when a command was rejected.
+    // A time point when it was rejected.
     google.protobuf.Timestamp when_rejected = 4;
 }
 
 // The rejection to create order.
 //
-// It is thrown if the order already exists.
+// It is thrown if an order already exists.
```

Comment:
Maybe we need to note there that it was attempt to create order with the same user, date and vendor as already exists.

Reply:
Is it better?
"Order for this user on the same date and for the same vendor already exists."